mdbenchmark/generate.py

```diff
@@ -245,6 +247,12 @@ def generate(name, cpu, gpu, module, host, min_nodes, max_nodes, time, skip_vali
                     time=time,
                 )
 
+            bundle = mds.discover(directory)
+            df = DataFrameFromBundle(bundle)
+            df_short = ConsolidateDataFrame(df)
```

Comment:
We need to rewrite this logic, so we first ask the user if they are OK with generating the following benchmarks (printing the summarized DataFrames and throwing a Yes/No prompt at them).

If yes, we actually generate the folders with `write_benchmark`, if no, we cancel and tell them that we did.

To print a summary of the DataFrames at once, you must create an empty list (e.g., `summary_dataframe`) outside of this loop (at the top `for m in module:`), append the `DataFrame` that is returned by `ConsolidateDataFrame(df)` here. After you finish with the `for-loop`, you can easily just print the result of `pd.concat(summary_dataframe)` at once.